packages/ws-server/src/controllers/index.ts

```diff
@@ -56,7 +56,10 @@ const handleSendingRequestsToRelay = async ({
 
   try {
     const resolvedParams = resolveParams(method, params);
-    const txRes = await relay.eth()[method.split('_')[1]](...resolvedParams, requestIdPrefix);
+    const [service, methodName] = method.split('_');
+
+    const txRes = await relay[service]()[methodName](...resolvedParams, requestIdPrefix);
```

Comment:
nit: worth a little comment and explanation here as this relies on a given class and method structure across files